.buildkite/pipeline.yml

```diff
@@ -26,8 +26,32 @@ steps:
         environment:
           - PYTHONDONTWRITEBYTECODE=1
 
+  - label: ":docker: :hadolint:"
```

Comment:
Maybe this could be a bit more descriptive, since hadolint isn't entirely obvious (I know platform currently isn't, but maybe it should be)?

```suggestion
  - label: ":docker: :hadolint: linting Dockerfiles"
```

Reply:
Okay I've pushed the suggested changes 👍